source/reference/minio-server/settings/notifications/kafka.rst

```diff
@@ -40,11 +40,11 @@ For example, the following commands set two distinct Kafka service endpoints as
       .. code-block:: shell
          :class: copyable
       
-         set MINIO_NOTIFY_KAFKA_ENABLE_PRIMARY="on"
-         set MINIO_NOTIFY_KAFKA_BROKERS_PRIMARY="https://kafka1.example.net:9200, https://kafka2.example.net:9200"
+         export MINIO_NOTIFY_KAFKA_ENABLE_PRIMARY="on"
```

Comment:
Is there a `windows` | `not windows` differentiator for this text?

Reply:
no, we don't produce windows docs for the Minio Server reference docs. They're linux only.